package.json

```diff
@@ -26,31 +26,31 @@
     "@types/globby": "^6.1.0",
     "@types/lodash": "^4.14.108",
     "@types/mocha": "^5.2.0",
-    "@types/node": "^8.10.11",
+    "@types/node": "^10.0.7",
     "@types/proxyquire": "^1.3.28",
-    "@types/pumpify": "^1.4.0",
-    "@types/sinon": "^4.3.1",
+    "@types/pumpify": "^1.4.1",
+    "@types/sinon": "^4.3.2",
     "@types/source-map-support": "^0.4.0",
     "@types/through2": "^2.0.33",
     "chai": "*",
-    "codecov": "~3.0.0",
-    "eslint": "^4.10.0",
-    "eslint-config-prettier": "^2.7.0",
-    "eslint-plugin-node": "^6.0.0",
-    "eslint-plugin-prettier": "^2.3.1",
-    "gts": "^0.5.4",
+    "codecov": "~3.0.2",
+    "eslint": "^4.19.1",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-node": "^6.0.1",
+    "eslint-plugin-prettier": "^2.6.0",
+    "gts": "^0.6.0",
```

Comment:
`gts@0.6.0` doesn't support Node 4 anymore.

Reply:
Which is fine! We were already running check as it's own step in CircleCI, so I just dumped the posttest npm script *wipes hands*.

Reply:
What about the developer workflow though? as a developer, shouldn't a style check run as part of the tests? I think you should edit the workflow rather than dumping the `posttest` script.

Reply:
Fixed!